src/components/Gradebook/index.jsx

```diff
@@ -280,10 +283,24 @@ export default class Gradebook extends React.Component {
   };
 
   handleClickExportGrades = () => {
+    trackEvent(() => ({
```

Comment:
I'd prefer for these `trackEvent` calls to be moved into the container (i.e. `GradebookPage/index.jsx`) and passed in to the component as new function props, which we can invoke here.